src/timer.rs

```diff
@@ -359,6 +426,14 @@ mod sealed {
         fn set_nchannel_polarity(channel: u8, p: Polarity);
     }
 
+    pub trait WithCcCommon: General {
+        const CC_CH_NUMBER: u8;
+        const CC_COMP_CH_NUMBER: u8;
+        fn read_cc_value(channel: u8) -> u32;
+        fn enable_channel(channel: u8, b: bool);
+        fn set_channel_polarity(channel: u8, p: Polarity);
+    }
+
```

Comment:
I do not see ant sense in additional copy of `WithPwmCommon` trait.
It is better to just rename `WithPwmCommon` to something more common.